system-addon/content-src/components/CollapsibleSection/_CollapsibleSection.scss

```diff
@@ -162,25 +162,21 @@
   }
 
   .section-disclaimer {
+    $min-button-height: 26px;
+
     color: $grey-60;
     font-size: 13px;
     margin-bottom: 16px;
     position: relative;
 
     .section-disclaimer-text {
       display: inline-block;
+      min-height: $min-button-height;
+      width: calc(100% - 126px);
 
-      @media (min-width: $break-point-small) {
+      @media (max-width: $break-point-medium) {
```

Comment:
ok, with this it matches the current behavior and the narrowest layout which i hope nobody ever uses. heh